setup.py

```diff
@@ -1,43 +1,63 @@
 from setuptools import setup
 
 setup(
-    name='beacon_network',
-    version='0.2.dev',
-    description='Beacon Network services',
-    long_description_content_type='text/markdown',
+    name="beacon_network",
+    version="0.2.dev",
+    description="Beacon Network services",
+    long_description_content_type="text/markdown",
     project_urls={
-        'Source': 'https://github.com/CSCfi/beacon-network',
+        "Source": "https://github.com/CSCfi/beacon-network",
     },
-    author='CSC - IT Center for Science',
+    author="CSC - IT Center for Science",
     classifiers=[
-        'Development Status :: 3 - Alpha',
-        'Intended Audience :: Developers',
-        'Topic :: Internet :: WWW/HTTP :: HTTP Servers',
-        'License :: OSI Approved :: Apache Software License',
-        'Programming Language :: Python :: 3.6',
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Developers",
+        "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3.6",
     ],
-    packages=['aggregator', 'aggregator/config', 'aggregator/endpoints',
-              'aggregator/utils', 'registry', 'registry/config',
-              'registry/endpoints', 'registry/schemas',
-              'registry/utils'],
-    package_data={'': ['*.json', '*.ini']},
+    packages=[
+        "aggregator",
+        "aggregator/config",
+        "aggregator/endpoints",
+        "aggregator/utils",
+        "registry",
+        "registry/config",
+        "registry/endpoints",
+        "registry/schemas",
+        "registry/utils",
+    ],
+    package_data={"": ["*.json", "*.ini"]},
     install_requires=[
-        'aiohttp', 'asyncpg', 'aiohttp_cors', 'uvloop',
-        'asyncio', 'aiocache', 'aiomcache', 'ujson',
-        'jsonschema==3.0.2', 'gunicorn'
+        "asyncio==3.4.3",
+        "aiohttp==3.7.4.post0",
+        "aiohttp-cors==0.7.0",
+        "aiocache==0.11.1",
+        "aiomcache==0.6.0",
+        "ujson==4.0.2",
+        "uvloop==0.14.0; python_version < '3.7'",
+        "uvloop==0.15.2; python_version >= '3.7'",
+        "asyncpg==0.22.0",
+        "jsonschema==3.2.0",
+        "gunicorn==20.1.0",
     ],
     extras_require={
-        'test': ['coverage', 'pytest<6.3', 'pytest-cov',
-                 'coveralls', 'testfixtures', 'tox',
-                 'flake8', 'flake8-docstrings', 'asynctest', 'aioresponses'],
-        'docs': [
-            'sphinx >= 1.4',
-            'sphinx_rtd_theme']
+        "test": [

```

Comment:
we should fix `test` and `docs` packages as well

Reply:
done